src/iap/shared/components/game-card/game-card.component.html

```diff
@@ -0,0 +1,18 @@
+<div class="iap-card">
+	<div class="iap-card__avatar"></div>
+	<div class="iap-card__info">
+		<div class="iap-card__availability">Available today!</div>
+
+		<div class="price-info">
```

Comment:
:point_right: :point_left: 
```suggestion
		<div class="iap-price-info">
```